website/docs/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -218,11 +230,13 @@ Properties without a default value are required.
 | `data.attributes.name`               | string | `null`  | An optional display name for the OAuth Client. If left `null`, the UI will default to the display name of the VCS provider.                                                                    |
 | `data.attributes.key`                | string |         | The OAuth Client key. It can refer to a Consumer Key, Application Key, or another type of client key for the VCS provider.                                                                     |
 | `data.attributes.http-url`           | string |         | The homepage of your VCS provider (e.g. `"https://github.com"` or `"https://ghe.example.com"` or `"https://gitlab.com"`).                                                                      |
-| `data.attributes.api-url`            | string |         | The base URL as per your VCS provider's API documentation (e.g. `"https://api.github.com"`, `"https://ghe.example.com/api/v3"` or `"https://gitlab.com/api/v4"`).                            |
+| `data.attributes.api-url`            | string |         | The base URL as per your VCS provider's API documentation (e.g. `"https://api.github.com"`, `"https://ghe.example.com/api/v3"` or `"https://gitlab.com/api/v4"`).                              |
 | `data.attributes.oauth-token-string` | string |         | The token string you were given by your VCS provider                                                                                                                                           |
 | `data.attributes.private-key`        | string |         | **Required for Azure DevOps Server. Not used for any other providers.** The text of the SSH private key associated with your Azure DevOps Server account.                                      |
 | `data.attributes.secret`             | string |         | The OAuth Client secret. For BitBucket Server, this secret is the the text of the SSH private key associated with your BitBucket Server Application Link.                                      |
 | `data.attributes.rsa-public-key`     | string |         | **Required for BitBucket Server in conjunction with the `secret`. Not used for any other providers.** The text of the SSH public key associated with your BitBucket Server Application Link.   |
+| `data.attributes.organization-scoped` | boolean |         | Whether or not the oauth client is scoped to all projects and workspaces in the organization. Defaults to `true`.
+| `data.relationships.projects.data[]` | array\[object] | `[]`      | A list of resource identifier objects that defines which projects are associated with the oauth client. This is only used when `data.attributes.organization-scoped` is set to false. These objects must contain `id` and `type` properties, and the `type` property must be `projects`. For example, `{ "id": "prj-AwfuCJTkdai4xj9w", "type": "projects" }`. |
```

Comment:
@robin-norwood based on my comment above, I would need to remove this sentence
```This is only used when `data.attributes.organization-scoped` is set to false.```

Reply:
Maybe reword it like:
```If `data.attributes.organization-scoped` is set to `false`, the OAuth client will only be available to the projects in this list.```